Dockerfile

```diff
@@ -86,14 +96,15 @@ ARG JX_RELEASE_VERSION=2.5.1
 ARG JENKINS_AGENT_VERSION=4.11.2-2
 ARG ASDF_VERSION=0.8.1
 
-LABEL io.jenkins-infra.tools="img,container-structure-test,git,make,hadolint,gh,nodejs,npm,blobxfer,jx-release-version,jenkins-agent"
+LABEL io.jenkins-infra.tools="img,container-structure-test,git,make,hadolint,gh,nodejs,npm,blobxfer,jx-release-version,jenkins-agent,netlify-deploy"
```

Comment:
Test are failing with the message `Error: label io.jenkins-infra.tools value img,container-structure-test,git,make,hadolint,gh,nodejs,npm,blobxfer,jx-release-version,jenkins-agent,netlify-deploy does not match expected value: img,container-structure-test,git,make,hadolint,gh,nodejs,npm,blobxfer,jx-release-version,jenkins-agent
`

can you update `cst.yml`?